src/Forms/Shared/SettingsPage.xaml

```diff
@@ -0,0 +1,124 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<TabbedPage
+    x:Class="ArcGISRuntime.SettingsPage"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:resources="clr-namespace:Forms.Resources;assembly=ArcGISRuntime"
+    mc:Ignorable="d">
+    <TabbedPage.Children>
+        <ContentPage Title="About">
+            <ContentPage.Content>
+                <WebView x:Name="AboutPage" />
+            </ContentPage.Content>
+        </ContentPage>
+        <ContentPage Title="Licenses">
+            <ContentPage.Content>
+                <WebView x:Name="LicensePage" />
```

Comment:
These don't look quite right on iOS. I wonder if a viewport meta tag would help? https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag I think that's done for the native iOS viewer.